docs/usage/features.md

```diff
@@ -55,7 +55,8 @@ option of nfd-worker.
 | **`cpu-pstate.scaling_governor`** | string | The value of the Intel pstate scaling_governor when in use, either 'powersave' or 'performance'.
 | **`cpu-cstate.enabled`**          | bool   | Set to 'true' if cstates are set in the intel_idle driver, otherwise set to 'false'. Unset if intel_idle cpuidle driver is not active.
 | **`cpu-rdt.<rdt-flag>`**          | true   | **DEPRECATED** [Intel RDT][intel-rdt] capability is supported. See [RDT flags](customization-guide.md#intel-rdt-flags) for details.
-| **`cpu-security.sgx.enabled`**    | true   | Set to 'true' if Intel SGX is enabled in BIOS (based a non-zero sum value of SGX EPC section sizes).
+| **`cpu-security.sgx.enabled`**    | true   | Set to 'true' if Intel SGX is enabled in BIOS (based on a non-zero sum value of SGX EPC section sizes).
+| **`cpu-security.sgx.epc`**        | int    | The total amount Intel SGX Encrypted Page Cache memory. It's only present if `sgx.enabled` is `true`.
```

Comment:
I'd say not expose this as a label by default. Just have it for NodeFeatureRules. WDYT?

Just realized this aspect of the tdx.total_keys and submitted #1130 to alter that.

Reply:
I thought about it too but did not get to addressing that yet. Previously, that "filtering" was done by `-resource-labels` and sources were free to create whatever labels they wanted. Thanks for #1130

Reply:
K, so maybe wait for #1130 to be merged and then rebase.

Reply:
sounds good. I already dropped this `features.md` addition.

Reply:
rebased